master/models.py

```diff
@@ -117,7 +117,7 @@ class Slum(models.Model):
     """Slum Database"""
     electoral_ward = models.ForeignKey(ElectoralWard)
     name = models.CharField(max_length=2048,blank=True,null=True)
```

Comment:
Remove blank and null for name in all the models administrative, electoral and slum. Aslo check the other fields for accepting nullable value